releasenotes/notes/amplitude-estimation-refactor-92dd0a2905b6fa0e.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    Add the refactored Amplitude Estimation algorithms.
+    The algorithms,
+    * ``AmplitudeEstimation`` (Brassard et al.)
+    * ``MaximumLikelihoodAmplitudeEstimation`` (Suzuki et al.)
+    * ``IterativeAmplitudeEstimation`` (Grinko et al.)
+    * ``FasterAmplitudeEstimation`` (Nakaji)
```

Comment:
```suggestion
    * :class:`~qiskit.algorithms.AmplitudeEstimation` (Brassard et al.)
    * :class:`~qiskit.algorithms.MaximumLikelihoodAmplitudeEstimation` (Suzuki et al.)
    * :class`~qiskit.algorithms.IterativeAmplitudeEstimation` (Grinko et al.)
    * :class:`~qiskit.algorithms.FasterAmplitudeEstimation` (Nakaji)
```

Reply:
Also do you want to add links to the papers here? Or are they in the api docs already (in which case I'd suggest removing them from the release notes and rely on the internal reference for the citation.